apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.YCSB/workload_template

```diff
@@ -0,0 +1,203 @@
+﻿# Copyright (c) 2012-2016 YCSB contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License. See accompanying
+# LICENSE file.
+
+# Yahoo! Cloud System Benchmark
+# Workload Template: Default Values
+#
+# File contains all properties that can be set to define a
+# YCSB session. All properties are set to their default
+# value if one exists. If not, the property is commented
+# out. When a property has a finite number of settings,
+# the default is enabled and the alternates are shown in
+# comments below it.
+#
+# Use of most explained through comments in Client.java or
+# CoreWorkload.java or on the YCSB wiki page:
+# https://github.com/brianfrankcooper/YCSB/wiki/Core-Properties
+
+# The name of the workload class to use
+workload=com.yahoo.ycsb.workloads.CoreWorkload
```

Comment:
Is this irrelevant for this code, or is it invoking Java somewhere that I'm not aware of? It would be useful to restrict this file to just the options that are relevant, IMO.

Reply:
its irrelevant.  this file started from a "real" config file for spanner YCSB (as in -- not-a-pretend-YCSB-package).
So I wanted to ensure it worked as is with all properties that java uses.
Most of these are not actually used.